src/languages/coffee.coffee

```diff
@@ -553,6 +553,28 @@ exports.CoffeeScriptParser = class CoffeeScriptParser extends parser.Parser
             # In the 'beginner' case of a simple method call with a
             # simple base object variable, let the variable be socketed.
             @csSocketAndMark node.variable.base, depth + 1, 0, indentDepth
+
+          if node.args.length is 0 and not node.do
+            variableBounds = @getBounds(node.variable)
```

Comment:
Add a comment that you are assuming an open parentheses immediately after the node.variable.